Sources/JobsPostgres/PostgresJobsQueue.swift

```diff
@@ -451,15 +468,56 @@ public final class PostgresJobQueue: JobQueueDriver, CancellableJobQueue, Resuma
             logger: self.logger
         )
     }
-    // TODO: maybe add a new column colum for attempt so far after PR https://github.com/hummingbird-project/swift-jobs/pull/63 is merged?
-    func updateJob(id: JobID, buffer: ByteBuffer, connection: PostgresConnection) async throws {
+    /// Helper func which to be use by a scheduled jobs
+    /// for performing job clean up based on a given set of policies
+    public func processDataRetentionPolicy() async throws {
```

Comment:
Given we are expecting people to set up a Job to call this, why don't we provide them with one.